agents/pysmurf_controller/pysmurf_controller.py

```diff
@@ -307,7 +307,8 @@ def check_state(self, session, params=None):
                     pysmurf_action = reg.pysmurf_action.get(**kw),
                     pysmurf_action_timestamp = reg.pysmurf_action_timestamp.get(**kw),
                     stream_tag = reg.stream_tag.get(**kw),
-                    last_update = time.time()
+                    last_update = time.time(),
+                    stream_id = cfg.stream_id,
```

Comment:
Should be added to the `session.data` example in the docstring.

Reply:
Good catch, thanks! Just added.